test/e2e/node/main.go

```diff
@@ -174,6 +174,7 @@ func startLightClient(cfg *Config) error {
 
 	rpccfg := rpcserver.DefaultConfig()
 	rpccfg.MaxBodyBytes = tmcfg.RPC.MaxBodyBytes
+	rpccfg.MaxBatchRequestNum = tmcfg.RPC.MaxBatchRequestNum
```

Comment:
I found I have forgotten to add MaxBatchRequestNum here. Is it right?

Reply:
You're right. I think using `rpcserver.DefaultConfig()` needs to add this.